benchmarks/README.md

```diff
@@ -0,0 +1,10 @@
+## Benchmarks
```

Comment:
Can you add steps so that another engineer can repeat the KPI results?

For example, the jupyter notebook assumes `DATA_FILE_PATH = "data/Finals_dataset/phase2_train.csv"`. Is there a separate step to download the data?

Reply:
I mentioned this in the docstrings. Let me know if I should add it to the notebook as well.